README.md

```diff
@@ -1,5 +1,4 @@
 # UnCSS Online
-<img src="https://travis-ci.org/pajasevi/UnCSS-Online.svg">
```

Comment:
Not sure how that happened (it wasn't there when I was commiting). At least I've markdown'd it. <3